enterprise/cmd/repo-updater/internal/authz/perms_syncer.go

```diff
@@ -258,6 +258,9 @@ func (s *PermsSyncer) maybeRefreshGitLabOAuthToken(ctx context.Context, acct *ex
 		break
 	}
 	if oauthConfig == nil {
+		log15.Warn("PermsSyncer.maybeRefreshGitLabOAuthToken, external service has no auth.provider",
+			"extSvcId", acct.ID,
```

Comment:
```suggestion
			"externalAccountID", acct.ID,
```

This is external account not external service?

Reply:
nice catch, thanks!